packages/swapper/src/swappers/NearIntentsSwapper/swapperApi/getTradeRate.ts

```diff
@@ -117,7 +143,7 @@ export const getTradeRate = async (
             chainSpecific: { pubkey },
             sendMax: false,
           })
-          return feeData.fast.txFee
+          return feeData.average.txFee
```

Comment:
Aligns with `evm.getFees()` which is what we use in `useEvmFees()` for quote time fees polling, using average there, so this will make fees closer.

Tbh, bit of a cat and mouse game here and we should really settle on a single  one (average vs. fast) across the board.